api/apiFunctions.test.tsx

```diff
@@ -0,0 +1,43 @@
+ import { loginAccount, logoutAccount } from './apiFunctions';
+import { account } from './config';
+
+describe('Auth Functions', () => {
+
+  jest.mock('./apiFunctions', () => ({
+    loginAccount: jest.fn()
```

Comment:
When we mock a function, we should mock its return. This should fail a lot of your tests because it's returning nothing.